playroom/use-scope.tsx

```diff
@@ -45,12 +45,14 @@ const usePlayroomState = () => {
 
 const useScope = (): {theme: Theme; colors: Colors} => {
     const theme = useTheme();
+    const screenSize = useScreenSize();
     const playroomState = usePlayroomState();
 
     return {
         theme,
         colors: theme.colors,
         ...playroomState,
+        ...screenSize,
```

Comment:
@pladaria, this allows us to create playrooms using `isMobile`, `isTabletOrBigger`, etc